lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -161,6 +172,23 @@ def partition_pdf(
 
             return deformable_layout
 
+    def _cleanup_tmp(self):
+        now = time.time()
+        dir = tempfile.gettempdir()
+        for ent in os.scandir(dir):
+            if ent.name.startswith(self.tmp_prefix):
+                try:
+                    st = ent.stat()
```

Comment:
what's ent?

Reply:
Short for dirent (struct dirent in C), which is short for directory entry.  The Python type is os.DirEntry.

Reply:
can I make you name the variable `entry` or something that in my head doesn't refer to a talking tree person?